### PR TITLE
docs(readme): add instructions for running from a cloned repo

### PR DESCRIPTION
**Pull request by paulosalvatore**
## Description

Added instructions to the README for running the project from a cloned repository.

## Changes
- Step-by-step instructions for cloning the repository.
- Commands to install dependencies.
- Commands to start the project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,15 +69,42 @@ You can install Langflow with pip:
...
-Or
 
-If you would like to install from your cloned repo, you can build and install Langflow's frontend and backend with:
+Then, run Langflow with:
+
+```shell
+python -m langflow run
+```
+
+# Running Langflow from a Cloned Repository
```
Comment (nicoloboschi) [inline README.md:79]: can you add an item to the toc?

File: README.md [hunk 2]
```diff
@@ -69,15 +69,42 @@ You can install Langflow with pip:
...
+
+If you prefer to run Langflow from a cloned repository rather than installing it via pip, follow these steps:
+
+1. **Clone the Repository**
+
+First, clone the Langflow repository from GitHub:
+
+```shell
+git clone https://github.com/langflow/langflow.git
```
Comment (nicoloboschi) [inline README.md:88]:
```suggestion
git clone https://github.com/langflow-ai/langflow.git
```

File: README.md [hunk 3]
```diff
@@ -69,15 +69,42 @@ You can install Langflow with pip:
...
+To build and install Langflow’s frontend and backend, use the following commands:
 
 ```shell
 make install_frontend && make build_frontend && make install_backend
 ```
 
-Then, run Langflow with:
+3. **Run Langflow**
+
+Once the installation is complete, you can run Langflow with:
```
Comment (nicoloboschi) [inline README.md:107]: Actually in this case the command should be prefixed by `poetry run`

**Review by nicoloboschi — CHANGES_REQUESTED** [review]

**Review by ogabrielluiz — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (paulosalvatore) [reply README.md:88]: My bad. Thanks.

File: README.md (hunk 1 above)
Comment (paulosalvatore) [reply README.md:80]: Done!

File: README.md (hunk 3 above)
Comment (paulosalvatore) [reply README.md:108]: Perfect! Just added.

**Review by nicoloboschi — APPROVED** [review]
LGTM

**Pull request merged**